lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -88,10 +97,21 @@ def steps
         @steps ||= step_nav[:steps]
       end
 
+      def step_navs_combined_list
```

Comment:
Perhaps you could split this refactoring out into a separate commit and prove that the existing tests still pass. Then it will be easier to focus on the changes that directly implement showing step navs on pages where they've been disabled.